src/DataType/UserRequests.ts

```diff
@@ -14,36 +14,39 @@ export type UserRequest = {
 };
 
 export class UserRequests {
-    localforage_instance?: typeof localforage;
+    __localforage_instance?: typeof localforage;
 
-    constructor() {
+    getLocalforageInstance() {
```

Comment:
Any reason this isn't a [proper getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get)? 

That way, we could keep the existing `this.localforage_instance` accesses as-is.

Reply:
Typescript wasn't happy about this.

Reply:
I think there was a warning about the target being to low.